recipes/gridgen/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export CFLAGS="-I$PREFIX/include $CFLAGS"
+export LDFLAGS="-L$PREFIX/lib"
+export CPPFLAGS="-I/$PREFIX/include $CPPFLAGS"
+
+pushd gridgen
+
+./configure --prefix=$PREFIX
+
+make
+make lib
+make shlib
```

Comment:
@phobson you took this farther than most of the previous users.  That is good enough to me!